reference_files/0.1.0/complex.yaml

```diff
@@ -3,179 +3,171 @@
+    !core/complex (nan+1.17549435082e-38j), !core/complex (inf+0j), !core/complex (inf+0j),
+    !core/complex (nan+nanj), !core/complex (nan+infj), !core/complex (nan-infj),
+    !core/complex (inf-3.40282346639e+38j), !core/complex (inf+3.40282346639e+38j),
+    !core/complex (inf+1.19209289551e-07j), !core/complex (inf+5.96046447754e-08j),
+    !core/complex (inf+1.17549435082e-38j), !core/complex (-inf+0j), !core/complex (-inf+0j),
+    !core/complex (nan+nanj), !core/complex (nan+infj), !core/complex (nan-infj),
+    !core/complex (-inf-3.40282346639e+38j), !core/complex (-inf+3.40282346639e+38j),
+    !core/complex (-inf+1.19209289551e-07j), !core/complex (-inf+5.96046447754e-08j),
+    !core/complex (-inf+1.17549435082e-38j), !core/complex (-3.40282346639e+38+0j),
+    !core/complex (-3.40282346639e+38+0j), !core/complex (nan+nanj), !core/complex (nan+infj),
+    !core/complex (nan-infj), !core/complex (-3.40282346639e+38-3.40282346639e+38j),
+    !core/complex (-3.40282346639e+38+3.40282346639e+38j), !core/complex (-3.40282346639e+38+1.19209289551e-07j),
+    !core/complex (-3.40282346639e+38+5.96046447754e-08j), !core/complex (-3.40282346639e+38+1.17549435082e-38j),
+    !core/complex (3.40282346639e+38+0j), !core/complex (3.40282346639e+38+0j), !core/complex (nan+nanj),
+    !core/complex (nan+infj), !core/complex (nan-infj), !core/complex (3.40282346639e+38-3.40282346639e+38j),
+    !core/complex (3.40282346639e+38+3.40282346639e+38j), !core/complex (3.40282346639e+38+1.19209289551e-07j),
+    !core/complex (3.40282346639e+38+5.96046447754e-08j), !core/complex (3.40282346639e+38+1.17549435082e-38j),
+    !core/complex (1.19209289551e-07+0j), !core/complex (1.19209289551e-07+0j), !core/complex (nan+nanj),
+    !core/complex (nan+infj), !core/complex (nan-infj), !core/complex (1.19209289551e-07-3.40282346639e+38j),
+    !core/complex (1.19209289551e-07+3.40282346639e+38j), !core/complex (1.19209289551e-07+1.19209289551e-07j),
+    !core/complex (1.19209289551e-07+5.96046447754e-08j), !core/complex (1.19209289551e-07+1.17549435082e-38j),
+    !core/complex (5.96046447754e-08+0j), !core/complex (5.96046447754e-08+0j), !core/complex (nan+nanj),
+    !core/complex (nan+infj), !core/complex (nan-infj), !core/complex (5.96046447754e-08-3.40282346639e+38j),
+    !core/complex (5.96046447754e-08+3.40282346639e+38j), !core/complex (5.96046447754e-08+1.19209289551e-07j),
+    !core/complex (5.96046447754e-08+5.96046447754e-08j), !core/complex (5.96046447754e-08+1.17549435082e-38j),
+    !core/complex (1.17549435082e-38+0j), !core/complex (1.17549435082e-38+0j), !core/complex (nan+nanj),
+    !core/complex (nan+infj), !core/complex (nan-infj), !core/complex (1.17549435082e-38-3.40282346639e+38j),
+    !core/complex (1.17549435082e-38+3.40282346639e+38j), !core/complex (1.17549435082e-38+1.19209289551e-07j),
+    !core/complex (1.17549435082e-38+5.96046447754e-08j), !core/complex (1.17549435082e-38+1.17549435082e-38j)]
```

Comment:
I'm not quite clear on how these changes got incorporated into this, but :+1: nonetheless.